plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogSkipArchive.java

```diff
@@ -82,6 +82,8 @@ protected QueryRunner createQueryRunner()
                 .setIcebergProperties(
                         ImmutableMap.<String, String>builder()
                                 .put("iceberg.catalog.type", "glue")
+                                // this is not required, since iceberg.glue.skip-archive is true by default, but verify that Guice modules load correctly
```

Comment:
unrelated

Reply:
It is related, is the comment unclear?

Reply:
Not applicable anymore after cleaning up configs